test/language/expressions/object/identifier-shorthand-implements-invalid-strict-mode.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2020 Rick Waldron. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: >
+  Object literal shorthands are only valid with identifier references,
+  not property names. (implements)
```

Comment:
this phrasing is weird.

Perhaps _Object literal shorthands are limited to valid identifier references. Future Reserved Words are disallowed in Strict Mode._